app/models/order.rb

```diff
@@ -0,0 +1,50 @@
+class Order < ActiveRecord::Base
+  has_many :order_items
+  belongs_to :user
+
+  validates :order_state, presence: true
+  validates :name, presence: true, on: :update
+  validates :email, presence: true, on: :update
+  validates :address, presence: true, on: :update
+  validates :city, presence: true, on: :update
+  validates :state, presence: true, on: :update
+  validates :zip, presence: true, on: :update
+  validates :card_name, presence: true, on: :update
+  validates :last_four_cc, presence: true, on: :update
+  validates :cc_cvv, presence: true, on: :update
+  validates :billing_zip, presence: true, on: :update
+  validates :cc_exp_month, presence: true, on: :update
+  validates :cc_exp_year, presence: true, on: :update
+
+  def already_ordered?(product, order_id)
```

Comment:
By naming this method with a question mark, your teammates will expect the result of this method to be either true or false. By potentially returning an item it could cause problems down the road for readability.